x-pack/docs/en/security/authentication/security-domain.asciidoc

```diff
@@ -124,13 +125,15 @@ When adding realms to a security domain, avoid authenticating with a newly-added
 
 Removing realms from a security domain can lead to unexpected behaviors
 and is not recommended.
-Resources created before the removal can be owned by different users depending on the resource type:
+Resources created or updated before the removal can be owned by different users depending on the resource type:
 
 - <<user-profile,User profiles>> are owned by the user for whom the profile was last
 <<security-api-activate-user-profile,activated>>.
 For users whose realms are no longer in the same domain as the owner user, a new user profile
 will be created for them next time the activate user profile API is called.
-- Resources such as API keys are owned by the user who originally created them.
+- An API key is owned by the user who originally <<security-api-create-api-key,created>> or last <<security-api-update-api-key,updated>> it.
+Users whose realms are no longer in the same domain as the API key's current owner won't be able to update the API key. Likewise, they won't be able to perform queries on the API key that rely on ownership (for example, calls to the <<security-api-get-api-key,get API key API>> with the `owner` parameter set to `true`).
```

Comment:
Happy to cut the "for example, calls to the <<security-api-get-api-key,get API key API>> with the `owner` parameter set to `true`" if it's TMI in this context

Reply:
Agreed, the specific API mentions feel out of place here.

> better establish what ownership means

Do you mean what ownership means for API keys, or more generally, what ownership means in the context of security domains?

I propose to go ahead in two parts:

For this PR, simply state:

> An API key is owned by the user who originally <<security-api-create-api-key,created>> or last <<security-api-update-api-key,updated>> it. Note that the original creator of an API key will lose ownership, if their realms are no longer in the same domain as those of the current API key owner.

In the future API key overview doc, discuss API key ownership in more detail.

WDYT?

Reply:
Yep. Let's define what is ownership separately and more generally since it is not only just for API keys.